ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -158,6 +158,25 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
     ORKPredefinedTaskOptionExcludeAudio = (1 << 7)
 } ORK_ENUM_AVAILABLE;
 
+/**
+ Values that identify the hand(s) to be used in an active task.
+ 
+ By default, the participant will be asked to use their most affected hand.
+ */
+typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskHandOption) {
+    /// Which hand to use is undefined
+    ORKPredefinedTaskHandOptionUndefined = 0,
```

Comment:
I would like to keep this as "undefined" because that's what it is. For the original implementation, our researchers discovered that the users were picking a different hand (mood, time of day, who knows) and we changed it to explicitly check both hands because which hand they were using was undefined (ie. unspecified).